### ISSUE TITLE
[dev.icinga.com #7951] Ensure that additional visual cues are available when text color differences are used to convey information

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1232]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/7951

**Created by elippmann on 2014-12-03 10:35:11 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2016-02-19 16:06:06 +00:00_)
Target Version: _Accessibility_
Last Update: _2016-02-19 16:06:06 +00:00 (in Redmine)_

---

http://www.w3.org/TR/WCAG20-TECHS/general.html#G183 wrote:
> The intent of this technique is to provide a redundant visual cue for users who may not be able to discern a difference in text color. Color is commonly used to indicate the different status of words that are part of a paragraph or other block of text or where special characters or graphics cannot be used to indicate which words have special status. For example, scattered words in text may be hypertext links that are marked as such by being printed in a different color. This technique describes a way to provide cues in addition to color so that users who may have difficulty perceiving color differences or have low vision can identify them. \[...\]

**Acceptance Criteria:**

-   Locate all instances where the color of text is used to convey information.
-   Check that any text where color is used to convey information is also styled or uses a font that makes it visually distinct from other text around it.
